server/services/config/conditions/README.md

```diff
@@ -0,0 +1,31 @@
+# Updating conditions
```

Comment:
instructions on how to update conditions

Reply:
excellent!